index.html

```diff
@@ -1,504 +1,404 @@
-<!DOCTYPE html>
-<html lang="en">
-
-<head>

```

Comment:
The previous main branch was deleted and the new branch was created. Merging main branch with new branch caused merge conflicts. I am not an advanced git user.